staging/src/k8s.io/client-go/util/cert/cert.go

```diff
@@ -53,6 +54,27 @@ type AltNames struct {
 	IPs      []net.IP
 }
 
+// RemoveDuplicateAltNames removes duplicate items in altNames.
+func RemoveDuplicateAltNames(altNames *AltNames) {
```

Comment:
please move this function and the unit tests to:

cmd/kubeadm/app/util/pkiutil/pki_helpers.go
cmd/kubeadm/app/util/pkiutil/pki_helpers_test.go

let's keep it to the kubeadm use case for now, and later if it's someone needs it we can move it to client-go.

Reply:
ok